classifiers/nlp/ZeroShotTFClassifier/__init__.py

```diff
@@ -0,0 +1,236 @@
+import numpy as np
+
+from typing import List, Optional
+
+from jina.executors.classifiers import BaseClassifier
+from jina.executors.decorators import batching, as_ndarray
+from jina.executors.devices import TFDevice
+
+
+class ZeroShotTFClassifier(TFDevice, BaseClassifier):
+    """
+    :class: `ZeroShotTFClassifier` wraps tensorflow transformers from
+        huggingface, and performs zero shot learning classification
+        for nlp data.
+    """
+
+    def __init__(
+        self,
+        labels: List[str],
+        pretrained_model_name_or_path: str = "distilbert-base-uncased",
```

Comment:
Could u try to use `single-quote` wherever possible (also in tests and comments and raise messages)